Lib/test/test_gdb.py

```diff
@@ -202,6 +202,8 @@ def get_stack_trace(self, source=None, script=None,
             'BFD: ',
             # ignore all warnings
             'warning: ',
+            # See #32962
+            'Python Exception',
```

Comment:
Hum, this is not a good idea to ignore bugs :-( Please revert this change.

My https://github.com/python/cpython/pull/7693 should fix this bug.